pique/style.css

```diff
@@ -2575,7 +2575,7 @@ body:not(.no-background-fixed) .site-footer {
 #pique-hero .pique-panel-content .entry-content {
 	text-align: center;
 }
-#pique-hero .pique-panel-content .entry-content p {
+#pique-hero .pique-panel-content .entry-content p:not(*) {
```

Comment:
```suggestion
:where(#pique-hero .pique-panel-content .entry-content) p {
```

This code changes the default paragraph alignment to `center` where as in the normal page it says left aligned. Is that expected?

<img width="322" alt="image" src="https://user-images.githubusercontent.com/1935113/190135644-35e259b8-3ecd-47af-b341-8444e8b68e71.png">

I suggest to lower the specificity using [:where](https://developer.mozilla.org/en-US/docs/Web/CSS/:where).

Reply:
Yeah. I didn't notice that code was changing the alignment for default paragraph blocks. Thank you for the heads up!